src/Components/Header.js

```diff
@@ -10,80 +10,84 @@ import { useTheme } from "@mui/material/styles";
 import MenuIcon from "@mui/icons-material/Menu";
 
 export default function Header(props) {
-  const classes = useStyles();
-  var theme = useTheme();
-  const matches = useMediaQuery(theme.breakpoints.up("900"));
-  const heading = useMediaQuery(theme.breakpoints.up("sm"));
+	const classes = useStyles();
+	var theme = useTheme();
+	const matches = useMediaQuery(theme.breakpoints.up("900"));
+	const heading = useMediaQuery(theme.breakpoints.up("sm"));
 
-  return (
-    <AppBar
-      position="fixed"
-      style={{
-        background: "#fff",
-        width: "100vw",
-        top: 0,
-        boxShadow: "none",
-        fontFamily: "Roboto",
-      }}
-    >
-      {/* <CssBaseline /> */}
-      <Toolbar>
-        <img src={"images/GDSC LOGO 1.svg"} height={"65px"} />
-        <Typography>
-          <div
-            className={classes.heading}
-            style={{ fontSize: heading ? 24 : 16, fontFamily: "Roboto" }}
-          >
-            <div>Google Developer Student Club</div>
-            <div
-              style={{
-                color: "#959595",
-                fontSize: "14.74px",
-                fontFamily: "Roboto",
-                paddingTop: "3px",
-              }}
-            >
-              ITM Gwalior
-            </div>
-          </div>
-        </Typography>
+	return (
+		<AppBar
+			position="fixed"
+			style={{
+				background: "#fff",
+				width: "100vw",
+				top: 0,
+				boxShadow: "none",
+				fontFamily: "Roboto",
+			}}>
+			{/* <CssBaseline /> */}
+			<Toolbar>
+				<img src={"images/GDSC LOGO 1.svg"} height={"65px"} />
+				<Typography>
+					<div
+						className={classes.heading}
+						style={{
+							fontSize: heading ? 24 : 16,
+							fontFamily: "Roboto",
+						}}>
+						<div>Google Developer Student Club</div>
+						<div
+							style={{
+								color: "#959595",
+								fontSize: "14.74px",
+								fontFamily: "Roboto",
+								paddingTop: "3px",
+							}}>
+							ITM Gwalior
+						</div>
+					</div>
+				</Typography>
```

Comment:
@Sunny-25 there has been added lot of spacing which was not required

Reply:
@Sunny-25 ?